src/flowchem/client/client.py

```diff
@@ -14,10 +14,28 @@
 
 
 class FlowchemDeviceListener(FlowchemCommonDeviceListener):
-    def _save_device_info(self, zc: Zeroconf, type_: str, name: str) -> None:
+    """
+    creat
```

Comment:
You can explain better what this class does.

Reply:
as below